Sources/AlamofireSessionRenewer/AlamofireSessionRenewer.swift

```diff
@@ -8,105 +8,170 @@
 import Foundation
 import Alamofire
 
-public typealias SuccessRenewHandler = (String) -> Void
-public typealias FailureRenewHandler = (Bool) -> Void
-public typealias RenewCredentialHandler = ((@escaping SuccessRenewHandler, @escaping FailureRenewHandler) -> Void)
+public typealias SuccessRenewHandler = (String) async -> Void
+public typealias FailureRenewHandler = (Bool) async -> Void
+public typealias RenewCredentialHandler = ((SuccessRenewHandler, FailureRenewHandler) async -> Void)
 
 /// This class is responsible for authentication credentials renewing process
```

Comment:
nit: let's replace the comment with:
```suggestion
/// That actor is responsible for authentication credentials renewing process
```